packages/fxa-content-server/server/lib/csp/blocking.js

```diff
@@ -38,10 +35,9 @@ module.exports = function(config) {
   const PAIRING_SERVER_HTTP = PAIRING_SERVER_WEBSOCKET.replace(/^ws/, 'http');
   const SENTRY_SERVER = 'https://sentry.prod.mozaws.net';
   // create a unique array of origins from survey urls
-  const SURVEYS = [
-    ...new Set(surveyList.map(s => getOrigin(s.url))),
-  ];
-  const surveysEnabledAndSet = (config.get('surveyFeature.enabled') && SURVEYS.length);
+  const SURVEYS = [...new Set(surveyList.map((s) => getOrigin(s.url)))];
+  const surveysEnabledAndSet =
+    config.get('surveyFeature.enabled') && SURVEYS.length;
```

Comment:
Argh how do these formatting changes even sneak in?!  I `git add -p` and checked the staged changes and everything.  😖